source/mongorestore/mongorestore-behavior-access-usage.txt

```diff
@@ -114,6 +114,14 @@ following limitations apply:
   - :option:`--oplogReplay <mongorestore --oplogReplay>`
   - :option:`--preserveUUID <mongorestore --preserveUUID>`
 
+.. note:: Rollback on the target cluster during a restore or import
+   
+   If you are restoring or importing data to a cluster that undergoes a
+   rollback during the restore or import process, you should delete all 
+   restored or imported data and redo the process from the beginning to
```

Comment:
You have a few instances of 'restore or import' in this content. Consider rewording so that the phrase only occurs once with 'transfer' or 'restore' alone used after to make the note more succinct.

Reply:
I appreciate all your help on this one, @kennethdyer ! Just made the changes – LMK if they look okay to you. Thank you!